src/main/resources/webgoat/templates/about.html

```diff
@@ -53,6 +53,7 @@ <h3 class="modal-title" id="myModalLabel">About WebGoat - Provided by the OWASP
                     <li>Doug Morato (Developer &amp; CI)</li>
                     <li>Bruce Mayhew (Developer)</li>
                     <li>Ren&eacute; Zubcevic (Developer)</li>
+		    <li>&Agrave;ngel Oll&eacute Bl&aacutezquez (Developer)</li>
```

Comment:
Perhaps you can format it with a few extra spaces. 
&aacute misses a ;

Reply:
&eacute also misses the ;

Reply:
Good catch, sorry my fault, fixed and rebased.